src/3D/Camera.cpp

```diff
@@ -189,47 +209,32 @@ void Camera::handleKeyboardInput(const SDL_Event& e)
 			_dwv = glm::vec3(0.0f, 0.0f, 0.0f);
 		}
 	}
-	else if (e.key.keysym.scancode == SDL_SCANCODE_A)
-	{
+	case SDL_SCANCODE_A:
 		_dv.x += (e.type == SDL_KEYDOWN) ? -movementSpeed : -_dv.x;
-	}
-	else if (e.key.keysym.scancode == SDL_SCANCODE_D)
-	{
+	case SDL_SCANCODE_D:
 		_dv.x += (e.type == SDL_KEYDOWN) ? movementSpeed : -_dv.x;
-	}
-	else if (e.key.keysym.scancode == SDL_SCANCODE_LCTRL)
-	{
+	case SDL_SCANCODE_LCTRL:
 		_dv.y += (e.type == SDL_KEYDOWN) ? -movementSpeed : -_dv.y;
-	}
-	else if (e.key.keysym.scancode == SDL_SCANCODE_SPACE)
-	{
+	case SDL_SCANCODE_SPACE:
 		_dv.y += (e.type == SDL_KEYDOWN) ? movementSpeed : -_dv.y;
-	}
-	else if (e.key.keysym.scancode == SDL_SCANCODE_Q)
-	{
+	case SDL_SCANCODE_Q:
 		_drv.y += (e.type == SDL_KEYDOWN) ? _movementSpeed : -_drv.y;
-	}
-	else if (e.key.keysym.scancode == SDL_SCANCODE_E)
-	{
+	case SDL_SCANCODE_E:
 		_drv.y += (e.type == SDL_KEYDOWN) ? -_movementSpeed : -_drv.y;
-	}
-	else if (e.key.keysym.scancode == SDL_SCANCODE_R)
-	{
+	case SDL_SCANCODE_R:
 		_drv.x += (e.type == SDL_KEYDOWN) ? _movementSpeed : -_drv.x;
-	}
-	else if (e.key.keysym.scancode == SDL_SCANCODE_V)
-	{
+	case SDL_SCANCODE_V:
 		_drv.x += (e.type == SDL_KEYDOWN) ? -_movementSpeed : -_drv.x;
```

Comment:
Be careful, without a break statement, the execution will flow through and execute the next line in the switch statement.
https://docs.microsoft.com/en-us/cpp/cpp/break-statement-cpp?view=msvc-170